.github/workflows/lighthouse.yml

```diff
@@ -37,10 +37,11 @@ jobs:
                       const globber = await glob.create('**/src/routes/**/+page.svelte');
                       const files = await globber.glob();
                       const urls = files
-                          .map(x => x.match(/routes\/([a-zA-Z\/\[\]]*?)\/?\+page\.svelte/)[1])
-                          .map(x => x.replace('[paperId]', '42').replace('[projectId]', '12'))
+                          .map(x => x.match(/routes\/([\(a-zA-Z\/\[\]\)]*?)\/?\+page\.svelte/)[1])
+                          .map(x => x.replace('[paperId]', '42').replace('[projectId]', '12').replace(/\([a-z\-]+\)\//g, ''))
```

Comment:
This is only for groups. I updated the regex to also match routes in parentheses. Because groups are only to organize your pages but aren't part of the URL of a sub-page, we remove the group name from the path.

Reply:
Ah right, I forgot the groups.